tests/component/test_trigger_properties.py

```diff
@@ -0,0 +1,115 @@
+"""Contains a collection of pytest tests that validates trigger properties."""
+
+from nidaqmx.constants import TriggerType
+from nidaqmx.error_codes import DAQmxErrors
+from nidaqmx.errors import DaqError
+from nidaqmx.task import Task
+
+
+def test__trigger__get_float_property__returns_value(ai_voltage_chan_task: Task):
+    """Test to validate getter for trigger property of float type."""
+    # Default Rate of the pulse width filter timebase in NI PCIe-6363 device is 0.0
+    assert ai_voltage_chan_task.triggers.start_trigger.dig_edge_dig_fltr_timebase_rate == 0.0
+
+
+def test__trigger__set_float_property__returns_assigned_value(ai_voltage_chan_task: Task):
+    """Test to validate setter for trigger property of float type."""
+    value_to_test = 2.505
+    ai_voltage_chan_task.triggers.start_trigger.dig_edge_dig_fltr_timebase_rate = value_to_test
+
+    assert (
+        ai_voltage_chan_task.triggers.start_trigger.dig_edge_dig_fltr_timebase_rate == value_to_test
+    )
+
+
+def test__trigger__reset_float_property__returns_initial_value(ai_voltage_chan_task: Task):
```

Comment:
Change all references to "initial_value" to "default_value".

Reason: "default value" is the standard DAQmx terminology, and "initial value" could refer to a value assigned by the Create Channel function.

Reply:
Renamed to _returns_default_value